packages/Checkbox/src/Checkbox.styles.js

```diff
@@ -9,8 +9,17 @@ const getLabelLeftPadding = (checkboxSize, hasLabel) =>
 
 const smallCheckboxSize = tokens.checkbox.sizeSm;
 const mediumCheckboxSize = tokens.checkbox.sizeMd;
-const largeCheckboxSize = tokens.checkbox.sizeLg;
 
+const mediumCheckboxGeneralStyle = {
+  checkBoxStyles: {
+    height: mediumCheckboxSize,
+    width: mediumCheckboxSize,
+  },
+  labelStyles: hasLabel => ({
+    minHeight: mediumCheckboxSize,
+    padding: `1px 0 0 ${getLabelLeftPadding(mediumCheckboxSize, hasLabel)}`,
```

Comment:
If you use the same top padding for `MEDIUM` and `LARGE` then the `LARGE` size is a bit off center:
<img width="199" alt="Screen Shot 2021-11-19 at 2 47 11 PM" src="https://user-images.githubusercontent.com/14944896/142700921-f46c941f-0fc6-44b8-b831-fc5f3c977276.png">